ToolForm.cs

```diff
@@ -109,7 +106,9 @@ private List<Room> GetAllRooms(Document doc)
         {
             List<Room> rooms = new List<Room>();
 
-            FilteredElementCollector collector = new FilteredElementCollector(doc);
+            Autodesk.Revit.DB.View view = doc.ActiveView;
```

Comment:
Seems like you are using the variable view only once. Remove this line and add doc.activeView as parameter in the next function. 

Instead of:
Autodesk.Revit.DB.View view = doc.ActiveView;
FilteredElementCollector collector = new FilteredElementCollector(doc, view.Id);

do:
FilteredElementCollector collector = new FilteredElementCollector(doc, doc.activeView.Id);